### DISCUSSION TITLE
Blog: should tags be lowercased?

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#4400] [Q&A]
If a user sets two tags

```yaml
tags:
  - tag1
  - Tag1
```

these tags would be considered different tags. IMHO, tags are never meant to be distinct by the case, so I think it makes sense to lowercase the tags and treat Tag1 and tag1 the same tag

**Comment by squidfunk** [comment]
Thanks for suggesting. IMHO, the tags plugin should never merge tags. Those might be typos or desired, and also, some users might want tags to be title case, some lowercase. Thus, there's no canonical way. What we could do to catch typos is what we did for the blog plugin with the [`categories_allowed`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#+blog.categories_allowed) option, which allows specifying an inclusion list of categories to be checked when building the site. If a category is not in the list, the build will fail.

Other than that, you might define all of your tags in title case or lowercase, there's no limitation imposed by the theme.
